src/app/modules/portal-core/services/library-service/library.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { LibraryConfig } from './library.config';
+import { HttpClient } from '@angular/common/http';
+import { environment } from 'src/environments/environment';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LibraryService {
+
+  constructor(private Http: HttpClient) { }
+
+
+  // To get the Filter List
+  getFilterList() {
```

Comment:
Are we using this service?

Reply:
No, i am not using this service , we are using common npm service